### PR TITLE
Allow omitting default values

### PR DESCRIPTION
**Pull request by berhoel**
I added option `no_default_values` with test as suggested in issue #55.

### CODE REVIEW

**Comment by gaborbernat** [comment]
Please never put an issue number into the description of a PR. Just put the change description the PR introduces.

File: roots/test-default-handling/index.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. sphinx_argparse_cli::
+  :module: parser
+  :func: main
+  :no_default_values: false
```
Comment (gaborbernat) [inline roots/test-default-handling/index.rst:4]: What's with the double negative here? Either make this not take an argument or drop the no prefix.

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: roots/test-default-handling/index.rst (hunk 1 above)
Comment (gaborbernat) [reply roots/test-default-handling/index.rst:4]: Also, add a changelog entry.

**Comment by berhoel** [comment]
I made the changes as requested in the above comment.

I hope it is OK to introduce a new version number in the change log.

Sorry for the error in the test input file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.9.1
```
Comment (gaborbernat) [inline CHANGELOG.md:5]: This is a feature so should be `1.10.0` per semver rules.

**Review by gaborbernat — CHANGES_REQUESTED** [review]

**Comment by berhoel** [comment]
Thank you for your review.

I changed the version number in CHANGELOG.md.

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**